doc/src/vpr/command_line_usage.rst

```diff
@@ -334,6 +342,12 @@ Use the options below to override this default naming behaviour.
 
     **Default:** :option:`circuit <circuit>`.route
 
+.. option:: --flat_place_file <file>
+
+    Path to input :ref:`flat placement file <vpr_flat_placement_file>`.
+
+    **Default:** :option:`circuit <circuit>`.route
```

Comment:
Currently it is not set up to look for a default.  Should I have it do so?  Since it isn't generated by an earlier vpr stage and is not the input circuit, I am not sure a default makes sense.

Reply:
In that case you should delete the "Default:**" line. Right now it says the default is a .route file.